noxfile.py

```diff
@@ -129,6 +130,7 @@ def default(session, install_extras=True):
     # Run py.test against the unit tests.
     session.run(
         "py.test",
+        "-n=auto",
```

Comment:
I think the default is the number of CPUs on the instance, right? I recall in bigframes we set this a bit higher than that and it seemed to help.

Reply:
I believe you are correct, that the default is the number of CPUs on the instance.